unit-tests/native/src/test/scala/scala/scalanative/libc/LocaleTest.scala

```diff
@@ -0,0 +1,91 @@
+package scala.scalanative.libc
+
+import org.junit.Test
+import org.junit.Assert._
+
+import scalanative.unsafe._
+import scalanative.libc.locale._
+
+class LocaleTest {
+  @Test def categoryConstantsReturnCorrectValue(): Unit = {
+    // check LC_* constants are unique
+    assertEquals(
+      6,
+      Set(
+        locale.LC_ALL,
+        locale.LC_COLLATE,
+        locale.LC_CTYPE,
+        locale.LC_MONETARY,
+        locale.LC_NUMERIC,
+        locale.LC_TIME
+      ).size
+    )
+  }
+  @Test def lconvInCLocale(): Unit = {
+    Zone { implicit z =>
+      locale.setlocale(locale.LC_ALL, toCString("C"))
+    }
+    import LConv.LConvOps
+    val clocale = locale.localeconv()
+    Zone { implicit z =>
+      assertEquals(
+        !toCString("."),
+        !clocale.decimal_point
+      )
+      assertEquals(
+        !toCString(""),
+        !clocale.thousands_sep
+      )
+      assertEquals(
+        !toCString(""),
+        !clocale.grouping
+      )
+      assertEquals(
+        !toCString(""),
+        !clocale.int_curr_symbol
+      )
+      assertEquals(
+        !toCString(""),
+        !clocale.mon_decimal_point
+      )
+      assertEquals(
+        !toCString(""),
+        !clocale.mon_thousands_sep
+      )
+      assertEquals(
+        !toCString(""),
+        !clocale.mon_grouping
+      )
+      assertEquals(
+        !toCString(""),
+        !clocale.positive_sign
+      )
+      assertEquals(
+        !toCString(""),
+        !clocale.negative_sign
+      )
+      // CHAR_MAX
+      assertEquals(
+        127,
```

Comment:
CHAR_MAX can be either a signed or unsiged value, here's a snipped of limits.h header file.  I believe that's the root of problems on Arm64 in the CI. I suggest adding CHAR_MAX to limits.scala or even better create a new one in clib project (current one is defined in posixlib)

```
/* Minimum and maximum values a `signed char' can hold.  */
#  define SCHAR_MIN     (-128)
#  define SCHAR_MAX     127

/* Maximum value an `unsigned char' can hold.  (Minimum is 0.)  */
#  define UCHAR_MAX     255

/* Minimum and maximum values a `char' can hold.  */
#  ifdef __CHAR_UNSIGNED__
#   define CHAR_MIN     0
#   define CHAR_MAX     UCHAR_MAX
#  else
#   define CHAR_MIN     SCHAR_MIN
#   define CHAR_MAX     SCHAR_MAX
#  endif
```

Reply:
Thanks! can I directly add limits.h in this PR as limits.h contains only constants macros? Or should I make another PR?

Reply:
Added limits.h in this PR.